Build/Projects/FrameworkReferences.definition

```diff
@@ -25,6 +29,7 @@
         Name="OpenTK"
         Path="ThirdParty/Dependencies/OpenTK.dll" />
     </Service>
+    <Reference Include="MonoGame.Framework.Core.Include" />
```

Comment:
An end user should not need the pcl reference assemblies. Since they are just that references, i.e shims with no code in them only the public api. Those "reference assemblies" should be replaced by the real ones from the mono framework. They are only really needed on a dev machine  for building. If that doesn't work then something is truly wrong with linux ;)

-----Original Message-----
From: "June Rhodes" <notifications@github.com>
Sent: ‎21/‎12/‎2015 12:41
To: "mono/MonoGame" <MonoGame@noreply.github.com>
Cc: "Dean Ellis" <dellis1972@googlemail.com>
Subject: Re: [MonoGame] Initial check-in of .Core solution (#4339)

In Build/Projects/FrameworkReferences.definition:
> @@ -25,6 +29,7 @@
>          Name="OpenTK"
>          Path="ThirdParty/Dependencies/OpenTK.dll" />
>      </Service>
> +    <Reference Include="MonoGame.Framework.Core.Include" />
That will pull in the MonoGame.Framework.Core.Include include project because the Linux platform isn't using PCLs here.
I'm personally still extremely reserved about making the Linux platform reference PCLs since this is still a huge mess across distros at the moment; the Linux platform is the reason Protobuild warns so strongly against using them, because the issues they cause can manifest themselves on end-user machines or developer machines, not build servers. Debian for example doesn't even package the PCL reference assemblies (along with an ancient version of Mono presumably), so users need to install these manually if a PCL is being targeted.
But @cra0zy seems to think it's fine so I don't know... but if I do run into issues relating to PCLs on Linux in MonoGame I will file issues here since it's not practical to be asking end-users to manually download ZIPs and install them into /usr/lib/mono or where-ever they're meant to go.
—
Reply to this email directly or view it on GitHub.